projects/aca-content/src/lib/components/search/search-libraries-results/search-libraries-results.component.scss

```diff
@@ -1,4 +1,4 @@
-@import 'mixins';
+@import 'projects/aca-content/src/lib/ui/mixins';
```

Comment:
I cannot find any working solution to import it as just `mixin` 

@DenysVuika maybe you would know how to handle this issue?